pkg/channel/distributed/controller/kafkachannel/reconciler.go

```diff
@@ -242,7 +244,17 @@ func (r *Reconciler) configMapObserver(configMap *corev1.ConfigMap) {
 	// environment will also need to be re-parsed here.
 
 	// Load the Sarama settings from our configmap, ignoring the eventing-kafka result.
-	saramaConfig, err := kafkasarama.MergeSaramaSettings(nil, configMap)
+
```

Comment:
Minor nit - looks like the comment on line 246 was abandoned - probably belongs down below on line 257

Reply:
+1